python/ray/air/_internal/remote_storage.py

```diff
@@ -1,24 +1,17 @@
-import fnmatch
 import os
```

Comment:
Actually what I meant was (1) let's not touch this file, (2) copy paste any methods we need into storage_context.py